pkg/liqonet/ipam/ipam_integration_test.go

```diff
@@ -0,0 +1,165 @@
+package ipam_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/util/slice"
+
+	"github.com/liqotech/liqo/pkg/consts"
+	"github.com/liqotech/liqo/pkg/liqonet/errors"
+	liqonetIpam "github.com/liqotech/liqo/pkg/liqonet/ipam"
+)
+
```

Comment:
Since the test leverage multiple components, I would add a comment to describe which is the test scenario, before the real assertions.